src/objects/SkinnedMesh.d.ts

```diff
@@ -1,15 +1,19 @@
-import { Geometry } from './../core/Geometry';
 import { Material } from './../materials/Material';
+import { MeshBasicMaterial } from './../materials/MeshBasicMaterial';
+
 import { Matrix4 } from './../math/Matrix4';
 import { Skeleton } from './Skeleton';
 import { Mesh } from './Mesh';
 import { BufferGeometry } from '../core/BufferGeometry';
 
-export class SkinnedMesh extends Mesh {
+export class SkinnedMesh<
+	TGeometry extends BufferGeometry,
+	TMaterial extends Material = MeshBasicMaterial
```

Comment:
```suggestion
	TMaterial extends Material | Material[] = MeshBasicMaterial
```

Reply:
@FMS-Cat why is it needed to have the assignment here? Not asking about the basic material in particular but also having `= Material | Material[]`?

Reply:
see the discussion: https://github.com/mrdoob/three.js/pull/18720/files#r406096587

Reply:
yeah no figured it out, i think the other PR is the same as this now